Source/settings/items.ts

```diff
@@ -10,6 +10,9 @@ module FullScreenPokemon {
             "gameStarted": {
                 "valueDefault": false
             },
+            "collectionKeys": {
+                "valueDefault": []
+            },
```

Comment:
In this design, consumers of StateHolder will need to know to add a `collectionKeys` value to the ItemsHoldr. That's risky and bound to cause confusion.

If possible, it would be better to have StateHolder handle this internally.